.typos.toml

```diff
@@ -0,0 +1,36 @@
+files.extend-exclude = ["pytket/docs/pytket-docs-theming/**"]
+
+
+[default.extend-identifiers]
+Aer = "Aer"
+AerBackend = "AerBackend"
+AerStateBackend = "AerStateBackend"
+AerUnitaryBackend = "AerUnitaryBackend"
+AerDensityMatrixBackend = "AerDensityMatrixBackend"
+aer = "aer"
+qiskit_aer = "qiskit_aer"
+anc = "anc"
+arange = "arange"
+aer_unitary_b = "aer_unitary_b"
+aer_b = "aer_b"
+noisy_aer_b = "noisy_aer_b"
+aer_state_b = "aer_state_b"
+args_ser = "arg_ser"
+IY = "IY"
+iy = "iy"
+PNGs = "PNGs"
+NDArray = "NDArray"
+pattern_nd = "pattern_nd"
+pn = "pn"
+
```

Comment:
If we're not going to add typos as a C.I. check it probably makes sense to remove this file.